RedHat-UX/red-hat-design-system
# fix(footer): fix demo icons

## What I did

1. ~~import all the `fab` icons into the importMap for 11ty (should really only be done on footer pages)~~
2. correct the icon names in demo markup
3. deprecated `--pfe-icon--color` in favor of using slotted selector to support pfe-icon 2.0

## Testing Instructions

1. check out `dp/components/footer/demo/`
2. Icons should load
3. The icons should be 24px
4. icons should gray

### Screenshot

![Screenshot 2023-01-11 at 10 26 28 AM](https://user-images.githubusercontent.com/3428964/211846165-df6f629b-3323-4ec7-b9c9-844188b8962a.png)


## Notes to Reviewers
~~don't look too closely into this one ;)
this should be reverted in favour of `<rh-icon>`, when it's ready~~ nvm this is pure :fire:
